proto/cronos/query.proto

```diff
@@ -86,4 +91,15 @@ message QueryParamsResponse {
     Params params = 1 [(gogoproto.nullable) = false];
 }
 
+// QueryPermissionsRequest is the request type for the Query/Permissions RPC method.
+message QueryPermissionsRequest {
+    string address = 1;
+}
+
+// QueryPermissionsResponse is the response type for the Query/Permissions RPC method.
+message QueryPermissionsResponse {
+    bool can_change_token_mapping = 1;
+    bool can_turn_bridge = 2;
```

Comment:
this is fine, but just for consideration whether this could return a more "extensible" field instead (e.g. a set/vec of permission names, or an integer that would have a bitvector interpretation of permissions)

Reply:
returning an integer is actually easier but I feel its more readable using this format (especially if its used just for display)